app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java

```diff
@@ -4,15 +4,15 @@
 
 public class ParkingModel implements ParkingContract.Model {
 
-    private int parkingAvailable = -1;
+    private String parkingAvailable = "-1";
```

Comment:
Here you can use a constant (PARKING_AVAILABLE_DEFAULT_VALUE for example). And it should be an int, why did you change it as string?

Reply:
because I was receiving it as a string from the fragment and I didn't want to parse it, but i already fixed

Reply:
As we talk, we leave this as int in the model (because is the **number** of available lots and maybe we are going to need to perform operations with this value), and let the presenter to do the casts we need for both, the model and the view.